source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -595,7 +603,7 @@ namespace
         createNodeConnections(doc->getNodes(), stage->getRootPrim());
 
         // Read look information
-        if (readOptions && readOptions->readLookInformation)
+        if (!readOptions || readOptions->readLookInformation)
```

Comment:
Also same logic as writeDocument. We default to reading looks when no options are provided.